study-builder/studies/rarex/study.conf

```diff
@@ -1087,6 +1288,63 @@
       "order": 1
     },
 
+    ##Proxy general information survey reminder
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "PARENTAL_GENERAL_INFORMATION",
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "templates": [
+          {
+            "emailTemplate": ${emails.proxyFinishGeneralInformationReminder},
+            "language": "en",
+            "isDynamic": false
+          }
+        ],
+        "linkedActivityCode": "PARENTAL_GENERAL_INFORMATION",
+        "pdfAttachments": []
+      },
+      "preconditionExpr": """user.studies["rarex"].forms["PARENTAL_CONSENT"].isStatus("COMPLETE")
+        || user.studies["rarex"].forms["CONSENT_ASSENT"].isStatus("COMPLETE")""",
```

Comment:
I think you can remove the `preconditionExpr` and just rely on the `cancelExpr`. Since there is a `delaySeconds`. Housekeeping will not run this event until the delay has expired.